content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -35,14 +35,21 @@ components still rely on Endpoints. For now, enabling Endpoint Slices should be
 seen as an addition to Endpoints in a cluster, not a replacement for them.
 {{< /note >}}
 
-As an alpha feature, Endpoint Slices are not enabled by default in Kubernetes.
-To enable them, the EndpointSlice feature gate will need to be enabled
-(`--feature-gates=EndpointSlice=true`).
+Endpoint Slices are considered a beta feature, but only the API is enabled by
+default. Both the Endpoint Slice controller and the usage of Endpoint Slices by
+Kube Proxy are not enabled by default. 
+
+The Endpoint Slice controler creates and manages Endpoint Slices in a cluster. It
+can be enabled with the Endpoint Slice feature gate on API Server and Controller
```

Comment:
```suggestion
can be enabled with the `EndpointSlice` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) on the API Server and the Controller
```
(better to write “kube-controller-manager” I think: if someone is implementing their own controller manager, bets about its command line flags are off anyway).

If you're rewriting this, you could also switch it from passive voice to active voice (“You can enable&hellip”) as recommended in the style guide.

Reply:
Good call, I've updated that now. I also added glossary tooltips for kube-apiserver and kube-controller-manager, but not sure if that's overkill here.